draft-ietf-ohai-ohttp.md

```diff
@@ -429,9 +429,17 @@ Change controller:
 
 # HPKE Encapsulation
 
-HTTP message encapsulation uses HPKE for request and response encryption.  An
-encapsulated HTTP message includes a binary-encoded HTTP message and no other
-content; see {{BINARY}}.
+HTTP message encapsulation uses HPKE for request and response encryption.
+
+An encapsulated HTTP request includes a timestamp and a binary-encoded HTTP
```

Comment:
Does the encapsulated request actually include a timestamp? The format doesn't seem to. If this is just referring to the fact that there should be a Date header, it seems misleading.

Reply:
Editing error on my part; I missed this one when I cribbed from #88.